pvr.vuplus/resources/language/resource.language.en_gb/strings.po

```diff
@@ -17,1013 +17,1298 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 ###################
+# settings labels #
 ###################
 
+#label: Connection - host
 msgctxt "#30000"
 msgid "Enigma2 hostname or IP address"
 msgstr ""
 
 #empty string with id 30001
 
+#label: Connection - streamport
 msgctxt "#30002"
 msgid "Streaming port"
 msgstr ""
 
+#label: Connection - user
 msgctxt "#30003"
 msgid "Username"
 msgstr ""
 
+#label: Connection - pass
 msgctxt "#30004"
 msgid "Password"
 msgstr ""
 
+#label-category: connection
 msgctxt "#30005"
 msgid "Connection"
 msgstr ""
 
+#label-group: General - Icons
 msgctxt "#30006"
 msgid "Icons"
 msgstr ""
 
+#label-group: General - Program Streams
 msgctxt "#30007"
 msgid "Program Streams"
 msgstr ""
 
+#label: General - iconpath
 msgctxt "#30008"
 msgid "Icon path"
 msgstr ""
 
+#label-group: General - Update Interval
 msgctxt "#30009"
 msgid "Update Interval"
 msgstr ""
 
 #empty string with id 30010
 
+#label: Timers - timerlistcleanup
 msgctxt "#30011"
 msgid "Automatic timerlist cleanup"
 msgstr ""
 
+#label: Connection - webport
 msgctxt "#30012"
 msgid "Web interface port"
 msgstr ""
 
+#label: Channels - zap
 msgctxt "#30013"
 msgid "Zap before channelswitch (i.e. for single tuner boxes)"
 msgstr ""
 
+#label: Channels - setprogramid
 msgctxt "#30014"
 msgid "Set program id for live channel or recorded streams"
 msgstr ""
 
+#label: General - updateint
 msgctxt "#30015"
 msgid "Update interval"
 msgstr ""
 
+#label: Channels - usegroupspecificnumbers
 msgctxt "#30016"
 msgid "Use bouquet specific channel numbers from backend"
 msgstr ""
 
+#label: Recordings - onlycurrent
 msgctxt "#30017"
 msgid "Use only the DVB boxes' current recording path"
 msgstr ""
 
+#label-category: general
+#label-group: Channels
 msgctxt "#30018"
 msgid "General"
 msgstr ""
 
+#label-category: channels
 msgctxt "#30019"
 msgid "Channels"
 msgstr ""
 
+#label-category: advanced
+#label-group: Connection - Advanced
 msgctxt "#30020"
 msgid "Advanced"
 msgstr ""
 
 #empty strings from id 30021 to 30022
 
+#label: Recordings - recordingpath
 msgctxt "#30023"
 msgid "Recording folder on the receiver"
 msgstr ""
 
+#label: Advanced - powerstatemode
 msgctxt "#30024"
 msgid "Send powerstate mode on addon exit"
 msgstr ""
 
+#label: Channels - tvgroupmode
 msgctxt "#30025"
 msgid "TV bouquet fetch mode"
 msgstr ""
 
+#label: Channels - onetvgroup
 msgctxt "#30026"
 msgid "TV bouquet 1"
 msgstr ""
 
+#label: General - onlinepicons
 msgctxt "#30027"
 msgid "Fetch picons from web interface"
 msgstr ""
 
+label: Connection - use_secure
```

Comment:
Here you forget the `#`, can be this in conflict by language read?

But to have a note where string is used is always good, thanks!